functional-test/src/test/groovy/org/rundeck/util/container/BaseContainer.groovy

```diff
@@ -364,7 +364,7 @@ abstract class BaseContainer extends Specification implements ClientProvider {
         }
 
         Execution execution = MAPPER.readValue(r.body().string(), Execution.class)
-        waitForExecutionStatus(execution.id as String, WaitingTime.EXCESSIVE)
+        waitForExecutionFinish(execution.id as String, WaitingTime.EXCESSIVE)
```

Comment:
We need to change the name of the wait function in this method's comments: https://github.com/rundeck/rundeck/pull/9314/files#diff-389e36f39f7e4b74b8e3e0c5e9996dafa659960d77d217cd06626c4bbb9bdaf4L353-L359.